asserts/ifacedecls.go

```diff
@@ -26,6 +26,8 @@ import (
 	"regexp"
 	"strconv"
 	"strings"
+
+	"github.com/snapcore/snapd/interfaces"
```

Comment:
this is problematic, given that asserts is one of the few bits of snapd really used as a library by other things we try to keep imports of other parts at a minimum, we should (re)define the interface here